src/bootstrap-dev.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
```

Comment:
These `bootstrap` files shouldn't be needed anymore if the `fec` cli and shared webpack config are use. Only the modules exposed in the `fec.config.js` should be needed. 

The `entry(-dev).js` files are also not really needed anymore, but the `fec` cli is looking for them still, so we can't remove those, but we can point the imports they include just to `AppEntry.js`.